Rakefile

```diff
@@ -4,7 +4,3 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
-
```

Comment:
Removing these removed the duplicate test runs.  Now the initializer in `rspec-rails` sets up the test suite to run just once.